shared/constants/permissions.ts

```diff
@@ -29,4 +29,45 @@ export const ConnectionPermission = Object.freeze({
   connection_permission: 'connection_permission',
 });
 
+// This configuration specifies permission weight thresholds used to determine which
+// permissions to show or hide on certain Snap-related flows (Install, Update, etc.)
+export const PermissionWeightThreshold = Object.freeze({
+  snapInstall: 3 as const,
+  snapUpdateApprovedPermissions: 2 as const,
+});
+
+// Specify minimum number of permissions to be shown, when abstraction is applied
+export const MinPermissionAbstractionDisplayCount = 3;
+
+// Specify number of permissions used as threshold for permission abstraction logic to be applied
+export const PermissionsAbstractionThreshold = 3;
+
+export const PermissionWeight = Object.freeze({
```

Comment:
These should probably be named as constants? Like `MIN_PERMISSION_ABSTRACTION_DISPLAY_COUNT`.

Reply:
I was following already used naming convention in that file. So, if I change it to uppercase with `_`, it would be different 🤔 

I can change these if you prefer or keep as is. @FrederikBolding can you provide your thoughts as well?

Reply:
Normally we (Snaps) do use uppercase and separate with `_` for constants like this, but it is not consistent across the extension codebase, so I'll leave it up to you @david0xd.

Not a blocker IMO

Reply:
I would prefer keeping the current state that matches what's already in these files in the extension. I think if we go that way, like the Snaps is, that we should refactor everything then 🤷‍♂️ 

Can we leave it be for now @Mrtenz ?